packages/react-aria-components/test/Switch.test.js

```diff
@@ -153,4 +155,13 @@ describe('Switch', () => {
     expect(label).toHaveAttribute('data-readonly');
     expect(label).toHaveClass('readonly');
   });
+
+  it('should render data- attributes only on the outer element', () => {
```

Comment:
it's fine to have both, but we could also have just checked this on the other test added above